YOSO/MainWindow.py

```diff
@@ -139,6 +139,7 @@ def __init__(self):
         self._next_image_action.triggered.connect(self.nextImage)
 
         menubar = self.menuBar()
+        menubar.setNativeMenuBar(False)
```

Comment:
This is again a very specific problem of a very specific environment. I'm not a Ubuntu user, but I'd expect global/app-specific configuration for workaround.